PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript.rbxmx

```diff
@@ -337,10 +339,14 @@ UserInputService.GamepadConnected:connect(function(gamepadEnum)
 	if CurrentControlModule and IsTouchDevice then
 		CurrentControlModule:Disable()
 	end
-	if isJumpEnabled and IsTouchDevice then TouchJumpModule:Disable() end
+	evaluateShowJumpModule()
 	
 	ControlModules.Gamepad:Enable()
 end)
+MasterControl.ControlStepped:connect(function() -- Must be done constantly because there is no Humanoid.StateEnabledChanged event
```

Comment:
We will need to fix this before shipping, lets add a new event to humanoid for this